utils/manifests/generateComponent.go

```diff
@@ -19,6 +19,10 @@ func GenerateComponents(ctx context.Context, manifest string, resource int, cfg
 		inputFormat = "json"
 	}
 	wd := filepath.Join(utils.GetHome(), ".meshery", "bin")
+	err := os.Mkdir(wd, 0750)
+	if err != nil && !os.IsExist(err) {
+		return nil, err
```

Comment:
Where do we have a MeshKit error with suggestion to verify file system permissions on failure?

Reply:
Details for the user on what do do in this case of this error would be good.

Reply:
Reminder...

Reply:
done